Javascript/integrateUppercase.js

```diff
@@ -0,0 +1,30 @@
+/*
+대문자로 통일
+대문자와 소문자가 같이 존재하는 문자열을 입력받아 대문자로 모두 통일하여 문자열을 출력 하는 프로그램을 작성하세요.
+- 입력설명
+첫 줄에 문자열이 입력된다. 문자열의 길이는 100을 넘지 않습니다. 
+- 출력설명
+첫 줄에 대문자로 통일된 문자열이 출력된다. 
+- 입력예제 1
+ItisTimeToStudy 
+- 출력예제 1
+ITISTIMETOSTUDY
+*/
+
+const solution = (str) => {
+  let result = "";
+  if (str.length > 100) throw new Error("문자열의 길이는 100을 넘지 않습니다.");
+  // 아스키 코드 차이 값 산출
+  const diff = "a".charCodeAt() - "A".charCodeAt();
+  for (s of str) {
+    if (s >= "a" && s <= "z") {
+      let asciiCode = s.charCodeAt();
```

Comment:
```suggestion
      const asciiCode = s.charCodeAt();
```
asciiCode 변수 변동이 발생하지 않으니 const가 더 좋을 것 같네요.

Reply:
넵 변경했습니다!